package.json

```diff
@@ -211,6 +218,9 @@
     "yarn-deduplicate": "^3.1.0"
   },
   "dependencies": {
+    "@antv/hierarchy": "^0.6.14",
```

Comment:
you don't need add dependencies.
Vite optimise warning appears if dependencies from the packages folder (redisinsight/ui/src/packages) have not been installed.
or you can exclude them in the ui vite.config.mjs
```
optimizeDeps: {
    exclude: [
      'react-json-tree',
      'redisinsight-plugin-sdk',
      'plotly.js-dist-min',
      '@antv/x6',
      '@antv/x6-react-shape',
      '@antv/hierarchy'
    ],
```

Reply:
I am a bit confused, why shouldn't we add them as dependencies, if the warning is caused by them not being installed?

Reply:
They aren't used in the RedisInsight application. 
They are a part of plugins (redisinsight/ui/src/packages). Each plugin is built separately from RedisInsight. Plugins dependencies are not used in the Redis Insight. Vite optimization doesn't see the difference and shows useless warning

Reply:
there is documentation about Plugins
https://github.com/RedisInsight/RedisInsight/wiki/Plugin-Documentation

Reply:
got it! THank you for the explanation. Cleared those as well.